source/includes/pipeline_config/_027-tracking-tool.md.erb

```diff
@@ -0,0 +1,22 @@
+<%
+json ='{
+  "tracking_tool": {
+    "type": "external",
+    "attributes": {
+      "link": "https://${ID}example.com",
```

Comment:
A really bad example :). Generally `${ID}` goes in the url, not the hostname. Would you like to use a real example from jira or bugzilla?